src/sentry/tagstore/snuba/backend.py

```diff
@@ -33,11 +33,14 @@ def __init__(self, dictionary):
         dictionary['id'] = 0
         self.__dict__ = dictionary
 
+    def __repr__(self):
+        return '<ObjectWrapper: {!r}>'.format(vars(self))
+
 
 @register(ObjectWrapper)
 class ObjectWrapperSerializer(Serializer):
     def serialize(self, obj, attrs, user):
-        return self.__dict__
+        return obj.__dict__
```

Comment:
This was causing this serializer to return an empty mapping because there are no instance attributes.